src/Server/HTTPHandler.cpp

```diff
@@ -580,6 +585,10 @@ void HTTPHandler::processQuery(
         session->makeSessionContext();
     }
 
+    NameValueCollection headers;
+    for (auto it = request.begin(); it != request.end(); ++it)
+        headers.set(it->first, it->second);
```

Comment:
where is this used?

Reply:
sry, I forgot to delete them. There are my debug code.